BlueprintUI/Tests/ElementContentTests.swift

```diff
@@ -99,6 +99,66 @@ class ElementContentTests: XCTestCase {
 
         XCTAssertEqual(callCount, 2)
     }
+
+    func test_forEachElement() {
```

Comment:
Once `SingleChildStorage` from https://github.com/square/Blueprint/pull/395 lands, need to update this test

Reply:
Tada!